charts/influxdb/values.yaml

```
@@ -262,17 +262,25 @@ backup:
   startingDeadlineSeconds: ""
   annotations: {}
   podAnnotations: {}
+  nodeSelector: {}
 
   ## Google Cloud Storage
-  # gcs:
+  gcs:
+    image:
+      repository: google/cloud-sdk
+      tag: alpine
   #    serviceAccountSecret: influxdb-backup-key
   #    serviceAccountSecretKey: key.json
   #    destination: gs://bucket/influxdb
```

Indentation should consistent with the rest of the file, these elements are on the same level as image (applies also to azure and s3):

```suggestion
    # serviceAccountSecret: influxdb-backup-key
    # serviceAccountSecretKey: key.json
    # destination: gs://bucket/influxdb
```